modules/security-container-content-universal.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * security/container_security/security-container-content.adoc
+
+[id="security-container-content-universal_{context}"]
+= Creating redistributable images with UBI 
+
+To create containerized applications, you typically start with a trusted base
+image that offers the components that are usually provided by the operating system.
+These include the libraries, utilities, and other features the application
+expects to see in the operating system's file system.
+
+Red Hat Universal Base Images (UBI) were created to encourage anyone building their
+own containers to start with one that is made entirely from Red Hat Enterprise
+Linux rpm packages and other content. These UBI images are updated regularly
+to keep up with security patches and free to use and redistribute with
+container images built to include your own software. 
+
+Search the
+link:https://catalog.redhat.com/software/containers/explore[Red Hat Ecosystem Catalog]
+to both find and check the health of different UBI images.
+As someone creating secure container images, you might
+be interested in these two general types of UBI images:
+
+* **UBI**: There are standard UBI images for RHEL 7 and 8 (`ubi7/ubi` and
```

Comment:
There are still some instances of RHEL in here, that show up in the OKD version of the docs. Can you just double check whether these are correct, or if they need to use the attribute to be Fedora? (Or add ifdefs to change out if just an attribute isn't correct. For example, I assume we couldn't do that here since it wouldn't be appropriate to say "Fedora 7").

Reply:
Done.